### ISSUE TITLE
State of Project

### ISSUE THREAD

**Issue by scarab5q** [ngortheone/org-rs#41]
Hi there,

it's been a while since I have been active at all around here due to my exams at university and moving to a new city, and starting a new job. I'm now ready to commit to an open-source project. I noticed that there hasn't been any commits in a while and was wondering if anyone was still planning to keep this project going at all? I'm happy to start working on some issues right away but would like to know where the current state of the project is? This is still a project I think is really worth doing and really worth getting right.

**Comment by ngortheone** [comment]
Hey @scarab5q 

Yes, the project has became stale because I lost the motivation for it. Despite over 500 stars there were no people interested in contributing and according to my estimation finishing it alone would take years for me. So I have moved on to smaller projects or projects with more active community around it.
I am doing opensource for pleasure and part of the pleasure is getting stuff done - something that was hard for me to get while working on `org-rs` (there is still so much to be done before even first alpha prototype) 

> if anyone was still planning to keep this project going at all?

That ultimately depends on everyone, including you. For now I personally not going to actively write code, but still open to other forms of contributions. And depends on how it goes - it may or may not change in the future.

> I'm happy to start working on some issues right away but would like to know where the current state of the project is?

Despite I am not actively writing code at the moment I can still help with directing your effort and testing. The `headline_parser` branch contains my most recent updates: There are 2 things going on in that branch. 
1.  While working on headline parser I refactored `cursor` into its own crate. The amount of code in `cursor` was growing and some things were becoming copy-pasty. So I decided to fix that as it is
a lower abstraction level for the rest of the code base and it should be solid. 
2. The work on headline parser itself - translating corresponding function from elisp

So if you want - take a look at that branch and try to finish the swing (currently things are a bit broken there but shouldn't be too hard to fix). I am happy to answer any questions that you might have in the process.


>  This is still a project I think is really worth doing and really worth getting right.

I agree. This was the reason why I did not archive the project and hoped for more people come and help.

**Comment by agathazeren** [comment]
_TL;DR: Call for a co-collaborator._

I figure this is a good place to bring up a problem I have. 

I have been working, and have started trying to get some prs merged (specifically #44 is ready for merging, and I have an unpushed changeset that depends on that, and both of these are directly or transitivity for getting tests set up for #42.) and I realized a problem, which is that there is no one to review the prs. The repository is currently set up to require a reviewer that is not the author. I think that this is a good thing, as a second pair of eyes is very important. (Case in point See #43). 

Ideally, the way to solve this problem is for another person to join as a collaborator. @ngortheone has said that they don't want to take this role, so I am posting here to ask if anyone would like to take this on. I can only spend so much of my time on this, so the review workload is likely to be relatively low. Even better if you are also working on the code (and I will be able to review).

@ngortheone I believe is the one who has permissions to grant collaborator permissions, so they will have the decision to add people.

**Comment by ngortheone** [comment]
I have elevated @asa-z and @scarab5q  to admins to unblock you from merging the code. Also feel free to invite other people who
 are interested!

**Comment by ngortheone** [comment]
As of today @appetrosyan is granted admin rights on the project.
@scarab5q and @scotttrinh and @asa-z access was revoked due to inactivity.
I continue to be a passive observer and play a backup admin role.

**Comment by appetrosyan** [comment]
@ngortheone Thank you. I suggest we close this issue.

**Comment by appetrosyan** [comment]
> I have been working, and have started trying to get some prs merged (specifically https://github.com/org-rs/org-rs/pull/44 is ready for merging, and I have an unpushed changeset that depends on that, and both of these are directly or transitivity for getting tests set up for https://github.com/org-rs/org-rs/pull/42.) and I realized a problem, which is that there is no one to review the prs. The repository is currently set up to require a reviewer that is not the author. I think that this is a good thing, as a second pair of eyes is very important. (Case in point See https://github.com/org-rs/org-rs/pull/43).

@agathazeren I have reviewed the PRs and approved the ones that are clear to merge.

**Comment by agathazeren** [comment]
@appetrosyan  I have also lost interest in this project, and don't plan on continuing to contribute in the near future. You are welcome to merge any work of mine that was left as a draft if you want (since you asked in a pr).

I wish you the best of luck with this project
